### PR TITLE
Re: 766 Fix Flow 0.32.0 annotations

### PR DESCRIPTION
**Pull request by montogeek**
Implements #716

### CODE REVIEW

File: src/builder/webpack/config.common.js [hunk 1]
```diff
@@ -41,4 +38,17 @@ export default (config: PhenomicConfig): WebpackConfig => {
...
+      ...defaultConfig,
+      ...hardSourceRecordsPath(cacheDir),
+      plugins: [
+        ...defaultConfig.plugins,
+        hardSourcePlugin(cacheDir, config),
+      ],
+    }
+  }
+
+  return defaultConfig
```
Comment (MoOx) [inline src/builder/webpack/config.common.js]: Can you avoid this big block and instead do something like `...config.cache ? hardSourceRecordsPath(cacheDir) : {},`?

Comment (montogeek) [inline src/builder/webpack/config.common.js]: Yes, we can!

**Comment by MoOx** [comment]
You work is good @montogeek. It's ANOTHER breaking change in webpack 2 latest beta.

Thanks for the effort!

**Comment by montogeek** [comment]
OMG! Thanks! :D

**Comment by MoOx** [comment]
Actually I was not careful enough, this bug was introduced by this PR :) Anyway we detected an improved error handling.

**Comment by montogeek** [comment]
What bug?

**Pull request merged**